week-2/G-conditionals-2/exercise-1.js

```diff
@@ -7,8 +7,22 @@
 */
 
 function negativeOrPositive(number) {
+  if(number > 1){
+    return true;
+  }
+  else{
+    return false;
+  }
 
 }
+function negativeOrPositive(number){
```

Comment:
You've defined the same function twice, this will cause unexpected results. Define one function that will return your expected results for each input.

Reply:
@LucyMac I Just noticed that, will make correction